app/controllers/taxons_controller.rb

```diff
@@ -38,4 +40,20 @@ def return_404
   def taxon
     @taxon ||= Taxon.find(request.path)
   end
+
+  def taxon_overview_and_child_taxons(taxon)
+    accordion_items = taxon.child_taxons
+    return [] if taxon.child_taxons.empty?
```

Comment:
Minor thing, but you could use `accordion_items` here, instead of `taxon.child_taxons`.